datapackage_pipelines_mojp/bagnowka/processors/download.py

```diff
@@ -27,29 +32,32 @@ def _get_resource(self):
             for item_data in all_docs:
                 new = all_docs[item_data]
                 doc = self.download(new)
-                logging.info("hello world")
+                logging.info("hello DOWNLOAD")
```

Comment:
there is no point to log the same string every row
think about which log info will be useful in the future, and if it's needed to log every row at all, or better to log once before / after the iterations